AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -61,12 +61,32 @@ define([
          */
         initialize: function () {
             this._super().initView();
-            $(window).on('fileDeleted.enhancedMediaGallery', this.reloadAdobeGrid.bind(this));
-            $(window).on('folderDeleted.enhancedMediaGallery', this.reloadAdobeGrid.bind(this));
+            $(window).on('fileDeleted.enhancedMediaGallery', function () {
+                this.updateIsDownloadedField();
+                this.reloadAdobeGrid();
+            }.bind(this));
+            $(window).on('folderDeleted.enhancedMediaGallery', function () {
+                this.updateIsDownloadedField();
+                this.reloadAdobeGrid();
```

Comment:
I'd update the displayed record for the preview as part of `reloadAdobeGrid` method.
Also, I think it's better to update the whole record for the preview instead of a single filed.